SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security
+
+If you believe you have found a security vulnerability in Slicer, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, send email to [slicer+security@discoursemail.com](mailto:slicer+security@discoursemail.com).
+
+You should receive a response within 24 hours.
+
+## Preferred Languages
+
+We prefer all communications to be in English.
```

Comment:
```suggestion
We prefer all communications to be in English.

## Supported Versions

You can access the most up-to-date Slicer packages through the official [Slicer download](https://download.slicer.org/) website.

| Version | Support Status     | Update frequency                                   |
| ------- | ------------------ | -------------------------------------------------- |
| 5.5.x   | :white_check_mark: | Preview: Continual integration of features & fixes |
| 5.4.x   | :white_check_mark: | Stable: Essential security fixes                   |
| < 5.4   | :x:                | Not supported                                      |

```

Reply:
Following @pieper suggestion, we should probably be removed the exact version and instead have:

| Version | Support Status     | Update frequency                            |
| ------- | ------------------ | ------------------------------------------- |
| Preview   | :white_check_mark: | Continual integration of features & fixes |
| Stable   | :white_check_mark: | Essential security fixes                   |

Reply:
Probably best that only the latest Slicer stable is supported and not older Slicer stable releases.

Reply:
And also based on a suggestion of @pieper, adding a disclaimer like the following may be sensible:

> There is no restriction on use, but Slicer is NOT approved for clinical use and the distributed application is intended for research use. Permissions and compliance with applicable rules are the responsibility of the user. For details on the license see [here](https://slicer.readthedocs.io/en/latest/user_guide/about.html#license).